README.md

```diff
@@ -1,6 +1,6 @@
 # react-logic-switch
 
-[![npm package][npm-badge]][npm]
+[![npm package][npm-badge]][npm]  [![Build Status](https://travis-ci.org/olayenca/react-logic-switch.svg?branch=master)](https://travis-ci.org/olayenca/react-logic-switch)
```

Comment:
Should this URL be updated to this repo?

Reply:
According to [this](https://docs.travis-ci.com/user/status-images/) yes, it auto updates. But this will have to change after I make the build as part of the elsevierlabs-os and not from my fork. I can remove it for now.